docs/api/plot.rst

```diff
@@ -119,10 +124,6 @@ roc
 ---
 .. autofunction:: sklearn_evaluation.plot.roc
 
-scores_distribution
--------------------
-.. autofunction:: sklearn_evaluation.plot.scores_distribution
-
```

Comment:
looks like this was removed by accident?

Reply:
I see what happened. you put this once inside CalibrationCurve. let's revert it to how it was before, having two plots in the same class will be confusing